package.json

```diff
@@ -25,7 +25,8 @@
     "react-icons": "^4.4.0",
     "react-scripts": "5.0.0",
     "react-xarrows": "^2.0.2",
-    "web-vitals": "^2.1.4"
+    "web-vitals": "^2.1.4",
+    "webpack": "^5.75.0"
```

Comment:
As explained here https://github.com/kbss-cvut/react-maintenance-planner/pull/92#discussion_r1049345839, it is not necessary.